src/wizard/wizard-step.component.js

```diff
@@ -0,0 +1,325 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.wizard.component:pfWizardStep
+ * @restrict E
+ *
+ * @description
+ * Component for rendering a Wizard step.  Each step can stand alone or have substeps.  This directive can only be used as a child of pf-wizard.
+ *
+ * @param {string} stepTitle The step title displayed in the header and used for the review screen when displayed
+ * @param {string} stepId  Sets the text identifier of the step
+ * @param {number} stepPriority  This sets the priority of this wizard step relative to other wizard steps.  They should be numbered sequentially in the order they should be viewed.
+ * @param {boolean} substeps Sets whether this step has substeps
+ * @param {boolean=} nextEnabled Sets whether the next button should be enabled when this step is first displayed
+ * @param {boolean=} prevEnabled Sets whether the back button should be enabled when this step is first displayed
+ * @param {string=} nextTooltip The text to display as a tooltip on the next button
+ * @param {string=} prevTooltip The text to display as a tooltip on the back button
+ * @param {boolean=} wzDisabled Disables the wizard when this page is shown
+ * @param {boolean} okToNavAway Sets whether or not it's ok for the user to leave this page
+ * @param {boolean} allowClickNav Sets whether the user can click on the numeric step indicators to navigate directly to this step
+ * @param {string=} description The step description (optional)
+ * @param {object} wizardData Data passed to the step that is shared by the entire wizard
+ * @param {function()=} onShow The function called when the wizard shows this step
+ * @param {boolean=} showReview Indicates whether review information should be displayed for this step when the review step is reached
+ * @param {boolean=} showReviewDetails Indicators whether the review information should be expanded by default when the review step is reached
+ * @param {string=} reviewTemplate The template that should be used for the review details screen
+ */
+angular.module('patternfly.wizard').component('pfWizardStep', {
+  require: {
+    wizard: '^pfWizard'
+  },
+  transclude: true,
+  bindings: {
+    stepTitle: '@',
+    stepId: '@',
+    stepPriority: '@',
+    substeps: '=?',
+    nextEnabled: '<?',
+    prevEnabled: '<?',
+    nextTooltip: '<?',
+    prevTooltip: '<?',
+    disabled: '@?wzDisabled',
+    okToNavAway: '<?',
+    allowClickNav: '<?',
+    description: '@',
+    wizardData: '=',
+    onShow: '=?',
```

Comment:
So it seems like for our callback functions we are using "=" or "<", should we be looking to using real output bindings "&".  On the other hand, I think if we do use "&" we have to change our API to use $event obj.